ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/AppLovinNativeAdapter.java

```diff
@@ -0,0 +1,49 @@
+package com.applovin.mediation;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.applovin.sdk.AppLovinSdk;
+import com.google.android.gms.ads.AdRequest;
+import com.google.android.gms.ads.mediation.MediationNativeAdapter;
+import com.google.android.gms.ads.mediation.MediationNativeListener;
+import com.google.android.gms.ads.mediation.NativeMediationAdRequest;
+
+public class AppLovinNativeAdapter
+        implements MediationNativeAdapter {
+    static final String TAG = AppLovinNativeAdapter.class.getSimpleName();
+
+    public static final String KEY_EXTRA_AD_ID = "ad_id";
+    public static final String KEY_EXTRA_CAPTION_TEXT = "caption_text";
+
+    @Override
+    public void requestNativeAd(final Context context,
+                                final MediationNativeListener mediationNativeListener,
+                                final Bundle serverParameters,
+                                final NativeMediationAdRequest nativeMediationAdRequest,
+                                final Bundle mediationExtras) {
+        if (nativeMediationAdRequest.isContentAdRequested()
```

Comment:
Why don't we allow content ads?

Reply:
But on iOS we do?

Reply:
We don't support content ads on either platform. On iOS we are supporting Unified and App Install ad formats. 

Unified is a new format added by AdMob which is not available on Android yet.